backend/src/main/java/com/example/backend/contoller/Listing_Controller.java

```diff
@@ -1,20 +1,20 @@
 package com.example.backend.contoller;
 
 import com.example.backend.model.Listing;
-import com.example.backend.service.ListingService;
+import com.example.backend.service.Listing_Service;
```

Comment:
Laut der Konvention: Ohne _ war auch in Ordnung gewesen. Gibt es da einen besonderen Grund dafür?

Reply:
Ne eigentlich nicht . Wird rausgenommen :)